src/main/takeDetection/__tests__/takeDetection.test.ts

```diff
@@ -0,0 +1,331 @@
+import { AssertionError } from 'assert';
+import { Word } from 'sharedTypes';
+import { InjectableTakeGroup } from 'main/editDelete/injectTakeInfo';
+import { findSentences, Sentence, findTakes } from '../takeDetection';
+import getSimilarityScore from '../sentenceSimilarity';
+
+function mockGetSentenceSimilarity(
+  sentenceOne: string,
+  sentenceTwo: string
+): number {
+  if (sentenceOne === 'Hello world!' && sentenceTwo === 'Hi there.') {
+    return 0.8;
+  }
+  if (sentenceOne === 'Hi there.' && sentenceTwo === 'All ok?') {
+    return 0.2;
+  }
+  if (sentenceOne === 'All ok?' && sentenceTwo === 'test.') {
+    return 0.4;
+  }
+  if (sentenceOne === 'test.' && sentenceTwo === 'assessment.') {
+    return 0.8;
+  }
+  if (sentenceOne === 'assessment.' && sentenceTwo === 'exam') {
+    return 0.8;
+  }
+  throw AssertionError;
```

Comment:
90% sure you have to instantiate the error to properly throw it - otherwise you're throwing a class